docs/ModelZoo.md

```diff
@@ -1,20 +1,21 @@
 # The DeepLabCut Model Zoo! 
 
-🦒 🐈 🐕‍🦺 🐀 🐁 🦡 🦦 🐏 🐫 🐆 🦓 🐖 🐄 🐂 🦖 🐿 🦍 🦥
+![image](https://images.squarespace-cdn.com/content/v1/57f6d51c9f74566f55ecf271/8957c690-4f27-4430-8581-4161fd58d052/68747470733a2f2f696d616765732e73717561726573706163652d63646e2e636f6d2f636f6e74656e742f76312f3537663664353163396637343536366635356563663237312f313631363439323337333730302d50474f41433732494f4236415545343756544a582f6b6531375a77644742546f646449.png?format=450w)
+
 
 ## 🏠 [Home page](http://modelzoo.deeplabcut.org/)
 
-Started in 2020 and expanded in 2022, the model zoo is four things:
+Started in 2020, expanded in 20222 with PhD student [Shaokai Ye et al.](https://arxiv.org/abs/2203.07436v1), and the first proper [SuperAnimal Foundation Models]() published in 2024 🔥, the Model Zoo is four things:
 
-- (1) a collection of models that are trained on diverse data across (typically) large datasets, which means you do not need to train models yourself
-- (2) a contribution website for community crowd sourcing of expertly labeled keypoints to improve models in part 1!
+- (1) a collection of models that are trained on diverse data across (typically) large datasets, which means you do not need to train models yourself, rather you can use them in your research applications.
+- (2) a contribution website for community crowd sourcing of expertly labeled keypoints to improve models! You can get involved here: [contrib.deeplabcut.org](https://contrib.deeplabcut.org/).
 - (3) a no-install DeepLabCut that you can use on ♾[Google Colab](https://colab.research.google.com/github/DeepLabCut/DeepLabCut/blob/master/examples/COLAB/COLAB_DLC_ModelZoo.ipynb), 
```

Comment:
I think the colab here is pointing to the "old modelzoo", which has been causing confusion to the users. SuperAnimal modelzoo does not require create_pretrained_project and we have our own video inference API for it.

Reply:
already fixed.